spec/models/news_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe(News, type: :model) do
+  let(:news) { create(:news) }
+
+  context 'when title field is too short' do
+    it { is_expected.to_not allow_values('No').for(:title) }
```

Comment:
why do you have this test? does not 
```ruby
context 'when title field is minimum length' do
    it { is_expected.to validate_length_of(:title).is_at_least(3) }
end
```
cover it?

Reply:
@okohuch I thought it is needed to test all cases. Will delete that test

Reply:
but it already covered all of them